libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_common.c

```diff
@@ -268,8 +270,12 @@ BTGattAdvertismentParams_t xAdvertisementConfigB =
     .ucName                       = { BTGattAdvNameShort,                  4},
     .bSetScanRsp                  = true,
     .ulAppearance                 = 0,
-    .ulMinInterval                = bletestsMAX_ADVERTISEMENT_INTERVAL / 2,
-    .ulMaxInterval                = bletestsMAX_ADVERTISEMENT_INTERVAL,
+    .ulMinInterval                = bletestsMIN_CONNECTION_INTERVAL,
+    .ulMaxInterval                = bletestsMAX_CONNECTION_INTERVAL,
+    .usMinAdvInterval             = bletestsMAX_ADVERTISEMENT_INTERVAL / 2,
+    .usMaxAdvInterval             = bletestsMAX_ADVERTISEMENT_INTERVAL,
+    .usMinAdvInterval             = 0,
```

Comment:
Are these the same two field members repeated?

Reply:
Fixed